src/dotnet/Chat.UI.Blazor/Components/HistoricalPlaybackPanel.razor

```diff
@@ -22,11 +22,14 @@
             </div>
             @* Stop button *@
             <div class="c-stop flex flex-row items-center">
-                <button @onclick="OnStop" class="text-secondary transition duration-300 ease-in-out hover:scale-110">
-                    <svg class="h-5 fill-current p-0.5" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg">
-                        <path d="M0 0h24v24h-24z"/>
-                    </svg>
-                </button>
+                <Button Class="text-secondary transition duration-300 ease-in-out hover:scale-110"
```

Comment:
Can we set a default transition like this to `.button` class? They always "blink" on hover, so...